packages/browser/src/integrations/globalhandlers.ts

```diff
@@ -236,8 +236,8 @@ function _enhanceEventWithInitialFrame(event: Event, url: any, line: any, column
   // event.exception.values[0].stacktrace.frames
   const ev0sf = (ev0s.frames = ev0s.frames || []);
 
-  const colno = isNaN(parseInt(column, 10)) ? undefined : column;
-  const lineno = isNaN(parseInt(line, 10)) ? undefined : line;
+  const colno = Number.isNaN(parseInt(column, 10)) ? undefined : column;
```

Comment:
`Number.isNaN` is not supported by IE11 😬 so we need to remove that rule and revert these, sadly, at least until V8!